virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -10,344 +10,282 @@ ms.service: windows-containers
 
-#### Example: Default HNS Naming Behavior
+--- @Jason: Is this syntax right?
+```
+# General syntax:
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.interface=<INTERFACE NAME> <NETWORK NAME> 
 
-To put the behavior of this naming option into context, the screen capture below demonstrates how a network is named by the HNS service when this naming option is *not* used. In this example, the name of the network, "MyTransparentNetwork" is visible to Docker, as shown via the `docker network ls` command. The name of the network is not visible, however, to the HNS service, as shown via the `Get-ContainerNetwork` Windows PowerShell command; instead, a large alphanumeric name for the network has been automatically generated by HNS.
+# Example:
+C:\> docker network create -d transparent -o com.docker.network.windowsshim.interface="Ethernet 2" myTransparent2
+```
 
-><figure>
-  <img src="media/SpecifyName_Capture.PNG">
-  <figcaption>Example: The name of a network is <i>not</i> specified to the HNS service. </figcaption>
-</figure>
+### Remember to specify *--subnet* and *--gateway* when using static IP assignment
+When using static IP assignment, you must first ensure that the *--subnet* and *--gateway* parameters are specified when the network is created. The subnet and gateway IP address should be the same as the network settings for the container host - i.e. the physical network.
 
-#### Example: Specifying the Name of a Network to the HNS Service
 
-When the `-o com.docker.network.windowsshim.networkname=<network name>` *is* used, on the other hand, the HNS service uses the specified name instead of a generated name. This behavior is demonstrated in the screen capture below.
+### DHCP IP assignment not supported with L2Bridge networks
+--- @Jason: This first says only static is supported then only says dynamic is supported. What's the take-home I need to try to capture on this one?
 
-><figure>
-  <img src="media/SpecifyName_Capture_2.PNG">
-  <figcaption>Example: The name of a network is specified to the HNS service using the `-o com.docker.network.windowsshim.networkname=<network name>` option.</figcaption>
-</figure>
+Only static IP assignment is supported with networks created using the l2bridge driver.
 
+> When using an l2bridge network on an SDN fabric, only dynamic IP assignment is supported. Reference the [Attaching Containers to a Virtual Network](https://technet.microsoft.com/en-us/windows-server-docs/networking/sdn/manage/connect-container-endpoints-to-a-tenant-virtual-network) topic for more information.
 
-### Bind a Network to a Specific Network Interface
+--- @Jason: So this means people need to create the networks on different interfaces, right? using the `-o com.docker.network.windowsshim.interface=<INTERFACE NAME>` that I mention above.
 
```

Comment:
HNS will try and use an "open" network interface which is not already attached to an external vSwitch. If for some reason there are multiple network adapters available and a user wants to select a specific one, they can do so with this option